ui/src/lib/gql/subscriptions/index.ts

```diff
@@ -0,0 +1,25 @@
+import { createClient } from "graphql-ws";
+import { useMemo } from "react";
+
+import { config } from "@flow/config";
+import { useAuth } from "@flow/lib/auth";
+
+export const useWsClient = () => {
```

Comment:
I think this should be setup as a provider like the other gql providers and wrap the app. If not, when we have multiple subscriptions on a single page we will be setting up multiple clients...